bash_completionsV2.go

```diff
@@ -176,6 +176,14 @@ __%[1]s_handle_completion_types() {
 __%[1]s_handle_standard_completion_case() {
     local tab=$'\t' comp
 
+    # Short circuit to optimize if we don't have descriptions
+    if [[ $out != *$tab* ]]; then
+        while IFS='' read -r comp; do
+            COMPREPLY+=("$comp")
+        done < <(IFS=$'\n' compgen -W "$out" -- "$cur")
```

Comment:
Why the `IFS=$'\n'` here?  We don't use it for the bash v1 script which uses similar loops.  Do we need to?

Reply:
It's there to protect spaces in individual completions:
```shellsession
$ compgen -W $'foo bar\nquux baz\n' -- f
foo
$ IFS=$'\n' compgen -W $'foo bar\nquux baz\n' -- f
foo bar
```

Reply:
Ok, seems better.  Thanks